src/java.base/share/classes/java/util/Formatter.java

```diff
@@ -1934,12 +1933,9 @@
 
     private IOException lastException;
 
-    private final char zero;
-    private static double scaleUp;
-
-    // 1 (sign) + 19 (max # sig digits) + 1 ('.') + 1 ('e') + 1 (sign)
-    // + 3 (max # exp digits) + 4 (error) = 30
-    private static final int MAX_FD_CHARS = 30;
+    // Non-unicode code point value used to mark zero as uninitialized
```

Comment:
Sorry for being anal here, but BOM is a "noncharacter", a valid Unicode code point. So "noncharacter" over "Non-unicode code point" seems appropriate.
http://www.unicode.org/faq/private_use.html#nonchar1